source/include/VirtualInstrument.h

```diff
@@ -246,17 +248,17 @@ class ClumpParseState
     InstructionAllocator* _cia;
     
     Int32           _argCount;
-    void*           _argPointers[kMaxArguments];
-    TypeRef         _argTypes[kMaxArguments];
-    
-    Int32           _argPatches[kMaxArguments];     // Arguments that need patching
+    std::vector<void*> _argPointers;
+    std::vector<TypeRef> _argTypes;
+
     Int32           _argPatchCount;
-    
-    PatchInfo       _patchInfos[kMaxPatchInfos];
+    std::vector<Int32> _argPatches;    // Arguments that need patching
```

Comment:
Nope, this is correct.  In addition to refactoring to use vectors, I moved some of the counts around to always be declared immediately before the vector they went with, so this might've caused the diff to confuse you.